### DISCUSSION TITLE
Question: Is there a debian package for ssh2-python?

### DISCUSSION THREAD

**Discussion by hakaishi** [ParallelSSH/ssh2-python#168] [General]
I'm considering to package one of my applications for Debian based systems. But in order to do that the proper way would be to add all the dependencies to the Debian repos first.

Is you package by any chance already in the Debian repos and if not, would you consider packaging it?

Depending on the workload needed, I might even consider to be the Debian Maintainer for this package, if needed. Maybe.

**Comment by pkittenis** [comment]
Hi,

Thanks for the interest. I am not aware of a Debian/Ubuntu system package for this library. I do not intend to make one personally.

If others want to make one and maintain it I would be happy to see that.
